multi_conv/multi_converter.py

```diff
@@ -73,22 +72,30 @@ def main():
                 print(f"Directory '{directory}' created successfully.")
 
     # Read links from the link_file
-    with open(link_file, 'r') as link_file:
-        links = [line.strip() for line in link_file]
+    with open(link_file, 'r') as file:
+        links = [line.strip() for line in file if line not in ['\n', '\r\n']]
 
     # Download videos
+    quality = input(" Select your video quality (high or low) > ")
     for link in links:
-        downloader(url=link)
+        downloader(url=link, quality=quality)
 
     # Get the list of downloaded MP4 files
-    mp4_files = [f for f in os.listdir(mp4_dir) if f.endswith('.mp4')]
+    mp4_names = [f for f in os.listdir(mp4_dir) if f.endswith('.mp4')]
+
+    # Write the mp3 titles in the name_file
+    with open(name_file, "w") as file:
+        for line in mp4_names:
+            print("line", line)
+            file.write(line+"\n")
```

Comment:
There should be a way to remove ".mp4"  to avoid saving  it as file_name.mp4.mp3. Kindly address for final merge.